layers/meta-resin-intel/recipes-core/initrdscripts/files/init-external-media.sh

```diff
@@ -43,14 +43,26 @@ read_args() {
     done
 }
 
-boot_external_rootfs() {
-    mkdir /.flasher_root
-    mount /dev/disk/by-label/flash-root /.flasher_root
+boot_rootfs() {
+   local _what=$1
+
+    echo "init-external-media.sh: Waiting for udev to populate /dev/disk/by-label/$_what... "
+    while true
+    do
+        if ls -A /dev/disk/by-label/$_what >/dev/null 2>&1; then
+            break
+        fi
+        sleep 1
+    done
+    echo "init-external-media.sh: Found $_what label."
+
+    mkdir /$_what
+    mount /dev/disk/by-label/$_what /$_what
     # Watches the udev event queue, and exits if all current events are handled
     killall "${_UDEV_DAEMON##*/}" 2>/dev/null
 
-    exec switch_root /.flasher_root /sbin/init ||
-        fatal "Couldn't switch_root, dropping to shell"
+    exec switch_root /$_what /sbin/init ||
+        fatal "init-external-media.sh: Couldn't switch_root, dropping to shell"
```

Comment:
I was under the impression that when the command was not in place, it will return to shell. Which after testing it's indeed not the case. I'll remove it altogether